src/main/java/eu/arrowhead/translationmanager/TranslationManagerSystemInfo.java

```diff
@@ -0,0 +1,323 @@
+
+import eu.arrowhead.common.Constants;
+import eu.arrowhead.common.SystemInfo;
+import eu.arrowhead.common.http.filter.authentication.AuthenticationPolicy;
+import eu.arrowhead.common.http.model.HttpInterfaceModel;
+import eu.arrowhead.common.http.model.HttpOperationModel;
+import eu.arrowhead.common.model.InterfaceModel;
+import eu.arrowhead.common.model.ServiceModel;
+import eu.arrowhead.common.model.SystemModel;
+import eu.arrowhead.common.mqtt.model.MqttInterfaceModel;
+
+@Component(Constants.BEAN_NAME_SYSTEM_INFO)
+public class TranslationManagerSystemInfo extends SystemInfo {
+
+	//=================================================================================================
+	// members
+
+	private SystemModel systemModel;
+
+	@Value(Constants.$ENABLE_BLACKLIST_FILTER_WD)
+	private boolean blacklistEnabled;
+
+	@Value(Constants.$FORCE_BLACKLIST_FILTER_WD)
+	private boolean forceBlackist;
+
+	@Value(TranslationManagerConstants.$ALLOW_DISCOVERY_FLAGS_WD)
+	private boolean allowDiscoveryFlags;
+
+	@Value(TranslationManagerConstants.$ENABLE_AUTHORIZATION_WD)
+	private boolean authorizationEnabled;
+
+	@Value(TranslationManagerConstants.$ENABLE_CUSTOM_CONFIGURATION_WD)
+	private boolean customConfigurationEnabled;
+
+	//=================================================================================================
+	// methods
+
+	//-------------------------------------------------------------------------------------------------
+	@Override
+	public String getSystemName() {
+		return Constants.SYS_NAME_TRANSLATIONMANAGER;
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	@Override
+	public SystemModel getSystemModel() {
+		if (systemModel == null) {
+			SystemModel.Builder builder = new SystemModel.Builder()
+					.address(getAddress())
+					.version(Constants.AH_FRAMEWORK_VERSION);
+
+			if (AuthenticationPolicy.CERTIFICATE == this.getAuthenticationPolicy()) {
+				builder = builder.metadata(Constants.METADATA_KEY_X509_PUBLIC_KEY, getPublicKey());
+			}
+
+			systemModel = builder.build();
+		}
+
+		return systemModel;
+	}
+
+	//-------------------------------------------------------------------------------------------------
+	@Override
+	public List<ServiceModel> getServices() {
+		final ServiceModel translationReport = new ServiceModel.Builder()
+				.serviceDefinition(Constants.SERVICE_DEF_TRANSLATION_REPORT)
+				.version(TranslationManagerConstants.VERSION_TRANSLATION_REPORT)
+				.metadata(Constants.METADATA_KEY_UNRESTRICTED_DISCOVERY, true)
+				.serviceInterface(getHttpServiceInterfaceForTranslationReport())
+				.serviceInterface(getMqttServiceInterfaceForTranslationReport())
+				.build();
+
+		final ServiceModel translationBridge = new ServiceModel.Builder()
+				.serviceDefinition(Constants.SERVICE_DEF_TRANSLATION_BRIDGE)
+				.version(TranslationManagerConstants.VERSION_TRANSLATION_BRIDGE)
+				.metadata(Constants.METADATA_KEY_UNRESTRICTED_DISCOVERY, true)
```

Comment:
Not sure this should be always `true`. When maintaining a Local Cloud, I would prefer to have the option to prevent consumers from initiating a translation bridge (especially when I have a cooperative cloud with ....ServiceOrchestration).